.github/commit_summarizer.yaml.txt

```diff
@@ -0,0 +1,17 @@
+name: GPT Commits summarizer
```

Comment:
GPT summary of [None](https://github.com/SampatSuthar/ml-demo/blob/8d6f43151d61cd79b56f97f6f5206c12ecd101ab/.github/commit_summarizer.yaml.txt#None) - [d2e715](https://github.com/SampatSuthar/ml-demo/blob/f4c31c4d6e1e13ad49b9c7c04ec99df1d76c930c/.github/commit_summarizer.yaml.txt#d2e715e39db0ed235fcb2ad065c23b2328a31106):
* Added a new action to the .github/commit_summarizer.yaml file
* The action will write a comment about every commit in a pull request, containing major points
* The action will run when the pull request is opened or synchronized
* The action will run on a Ubuntu-latest environment
* The action will use the KanHarI/gpt-commit-summarizer@master package
* The action will use the Github token and OpenAI API key stored in the repository secrets